components/progressbar/metadata/meter.yml

```
@@ -1,34 +1,107 @@
 name: Meter
-description: Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+description: |
+  Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+  Meter should be only used with `S` or `L` t-shirt sizes along with Progress Bar classes of the same t-shirt size. [The Meter table of options](https://spectrum.adobe.com/page/meter/#Table-of-options) provides further details on which variants are available.
```

```suggestion
  Meter should be only used with `S` or `L` t-shirt sizes. [The Meter table of options](https://spectrum.adobe.com/page/meter/#Table-of-options) provides further details on which variants are available.
```

Is this accurate? It should just have a `spectrum-ProgressBar` base class right? Not `spectrum-Progressbar--sizeS` since the Meter t-shirt size class will be used.

---

yeah this could be confusing. Let's remove that statement.